onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2691,6 +2694,109 @@ ORT_API_STATUS_IMPL(OrtApis::Graph_GetParentNode, _In_ const OrtGraph* graph, _O
   API_IMPL_END
 }
 
+ORT_API_STATUS_IMPL(OrtApis::Graph_GetSubGraph, _In_ const OrtGraph* src_graph,
+                    _In_ const OrtNode** nodes,
+                    _In_ size_t num_nodes,
+                    _In_ bool to_create_standalone_sub_graph,
+                    _In_ bool copy_in_memory_initializer,
+                    _Outptr_ OrtGraph** dst_graph) {
+  API_IMPL_BEGIN
+
+  if (num_nodes == 0) {
+    return OrtApis::CreateStatus(ORT_INVALID_ARGUMENT, "'num_nodes' argument should be > 0");
+  }
+
+  if (to_create_standalone_sub_graph) {
+#if !defined(ORT_MINIMAL_BUILD)
+    std::unique_ptr<Model> model;
+    ep_graph_utils::GetSubGraphAsModelFromGraph(src_graph, nodes, num_nodes, copy_in_memory_initializer, model);
```

Comment:
Should this handle the potential error status returned by `GetSubGraphAsModelFromGraph`?